src/utils/interface.js

```diff
@@ -17,6 +15,29 @@ export function chat(chatMessage) {
   }
 }
 
+let AUTONOMOUS = true;
+
+export function isAutonomous() {
+  return AUTONOMOUS;
+}
+
+export function onKeyDown(event) {
+  // If letter pressed is Q, toggle autonomous controls
+  if (event.keyCode === 81) {
+    AUTONOMOUS = !AUTONOMOUS;
+    tagpro.sendKeyPress('up', true);
+    tagpro.sendKeyPress('down', true);
+    tagpro.sendKeyPress('left', true);
+    tagpro.sendKeyPress('right', true);
+    if (AUTONOMOUS) {
+      chat('Autonomy Mode updated: now AUTONOMOUS!');
+    } else {
+      chat('Autonomy Mode updated: now MANUAL!');
```

Comment:
I know you didn't write this but can you make this one line, using string templates like the console log below?